pelican/__init__.py

```diff
@@ -101,6 +101,12 @@ def _handle_deprecation(self):
                             'PAGE_LANG_URL'):
                 logger.warning("%s = '%s'", setting, self.settings[setting])
 
+        if self.settings.get('AUTORELOAD_IGNORE_CACHE'):
+            logger.warning('Found deprecated `AUTORELOAD_IGNORE_CACHE` in '
+                           'settings. It has no effect now and can be safely '
+                           'removed.')
```

Comment:
Umm, why? People can still enable cache and might want to disable it in ``--autoreload``.